packages/ui/src/components/vuestic-components/va-modal/VaModal.vue

```diff
@@ -94,7 +94,7 @@ import VaIcon from '../va-icon'
 
 class ModalProps {
   modelValue = prop<boolean>({ type: Boolean, default: false })
-  target = prop<string>({ type: String, default: '#app' })
+  parentNode = prop<string>({ type: String, default: 'body' })
```

Comment:
You can declare type above 
```ts
declare type CSSSelector = string
```

```suggestion
  parentNode = prop<CSSSelector | 'parent'>({ type: String, default: 'body' })
```

This will be useless for typing, but it would be easier to understand this prop for humans.